target-platforms/packages/portal-base/src/build.js

```diff
@@ -105,7 +127,59 @@ function runBundler() {
 
 	print(info`Running {liferay-npm-bundler}...`);
 
-	childProcess.spawnSync('node', [bundlerPath], {
-		stdio: 'inherit',
+	const {error, signal, status} = childProcess.spawnSync(
+		'node',
+		[bundlerPath],
+		{
+			stdio: 'inherit',
+		}
+	);
+
+	if (error) {
+		abort(error);
+	}
+
+	if (signal) {
+		abort(`{liferay-npm-bundler} received signal: ${signal}`);
+	}
+
+	if (status !== 0) {
+		abort(`{liferay-npm-bundler} exited with status: ${status}`);
+	}
+}
+
+function runSass() {
+	const scssFiles = findFiles(srcDir, (dirent) => {
```

Comment:
I'm not super familiar with the word `dirent`. Is this a thing or a typo?

Reply:
Nope, [it does exist](https://nodejs.org/api/fs.html#fs_class_fs_dirent) (I had to read the docs when giving it a name; I usually named it `item`, but this is apparently a better name :sweat_smile: ).